### PR TITLE
🐛 Bug: denmulderboomteelt.com cookie popup fix

### LINKED ISSUE #177446: www.denmulderboomteelt.com
### Issue URL (Incorrect Blocking)
 [https://www.denmulderboomteelt.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.denmulderboomteelt.com%2F)
Username: `@Morku`

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 1 with AdGuard disabled</summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard for Windows v7.17.0 (4705)
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode options: | Strip URLs from tracking parameters, <br>Hide your search queries, <br>Send Do-Not-Track header, <br>Hide your Referrer from third-parties, <br>Hide your User-Agent, <br>Remove X-Client-Data header from HTTP requests, <br>Protect against DPI, <br>Disable cache for third-party requests, <br>Disable WAP Push Message Routing Service, <br>Disable Windows Defender automatic samples submission, <br>Disable Windows Telemetry, <br>Turn off Advertising ID
DNS filtering: | server: `AdGuard`, `DNS`, `Non-filtering`<br>filters: `AdGuard Home with AdGuard DNS filter`
Filters: | <b>Ad Blocking:</b><br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Cookie Notices, <br/>AdGuard Popups, <br/>AdGuard Mobile App Banners, <br/>AdGuard Other Annoyances, <br/>AdGuard Widgets<br/><br/><b>Language-specific:</b><br/>AdGuard German
Userscripts: | https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js

### PR DESCRIPTION
**Pull request by Coolhabit**
Fix https://github.com/AdguardTeam/AdguardFilters/issues/177446

### CODE REVIEW

File: AnnoyancesFilter/Popups/sections/popups_allowlist.txt
Comment (AdamWr) [inline AnnoyancesFilter/Popups/sections/popups_allowlist.txt:51]: Is something broken when this cookie notice is hidden (I didn't notice any breakage when it's hidden, but maybe I have missed somthing)?
If not, then please add a rule to hide it, to - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Cookies/sections/cookies_specific.txt

If something is broken, then please move rules to - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt

**Review by AdamWr — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/Popups/sections/popups_allowlist.txt
Comment (Coolhabit) [reply AnnoyancesFilter/Popups/sections/popups_allowlist.txt]: move rules as you said, cookie-notice is needed

Comment (Coolhabit) [reply AnnoyancesFilter/Popups/sections/popups_allowlist.txt]: @AdamWr do I need to transfer it to the top without comment with link to task?

Comment (AdamWr) [reply AnnoyancesFilter/Popups/sections/popups_allowlist.txt]: We usually add new rules at the top, somewhere here - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt#L17-L20
so please move it there, but comment with link to issue should be added.

By the way, could you please write what exactly is broken when cookie notice is hidden?

Comment (AdamWr) [reply AnnoyancesFilter/Popups/sections/popups_allowlist.txt]: And one more question, why this exception rule:
```adblock
@@||v1.widget.futy.io/js/futy-widget-$script,domain=denmulderboomteelt.com
```
is required?
It's not blocked on my end, but maybe I have missed something.

Comment (Coolhabit) [reply AnnoyancesFilter/Popups/sections/popups_allowlist.txt]: you are right, it is unnecassary

Comment (Coolhabit) [reply AnnoyancesFilter/Popups/sections/popups_allowlist.txt]: `By the way, could you please write what exactly is broken when cookie notice is hidden?
`
Cookie popup is not opening without this rule

Comment (AdamWr) [reply AnnoyancesFilter/Popups/sections/popups_allowlist.txt]: So, maybe it can be just hidden, for example:
```adblock
denmulderboomteelt.com###cookie-notice-wrapper
```

Comment (Coolhabit) [reply AnnoyancesFilter/Popups/sections/popups_allowlist.txt]: @AdamWr not working like that
just in combination, that into the PR

moved rules to the top

Comment (AdamWr) [reply AnnoyancesFilter/Popups/sections/popups_allowlist.txt]: I mean just this rule:
```adblock
denmulderboomteelt.com###cookie-notice-wrapper
```
without adding exceptions.

Comment (Coolhabit) [reply AnnoyancesFilter/Popups/sections/popups_allowlist.txt]: @AdamWr In that case the cookie banner is not visible at all, including the background.
Is this good?

Comment (AdamWr) [reply AnnoyancesFilter/Popups/sections/popups_allowlist.txt]: The main purpose of AdGuard Cookie Notices filter/AdGuard Annoyances filter is to hide/block these cookie notices/popups.
We unblock them only if website functionality is broken when cookies are not accepted/rejected.
If there is no breakage when cookie notice is hidden then it's preferred solution.

Comment (Coolhabit) [reply AnnoyancesFilter/Popups/sections/popups_allowlist.txt]: Ok, done as you said, all works fine
@AdamWr

Comment (AdamWr) [reply AnnoyancesFilter/Popups/sections/popups_allowlist.txt]: Sorry, I should mention it, but it should be added to `cookies_specific.txt`.
There is already similar rule, so please add a domain to it:
https://github.com/AdguardTeam/AdguardFilters/blob/ff067d646e6ff9f9ba2451745dd187f05a30c29f/AnnoyancesFilter/Cookies/sections/cookies_specific.txt#L8064
and it this case adding a comment is not necessary.

Comment (Coolhabit) [reply AnnoyancesFilter/Popups/sections/popups_allowlist.txt]: @AdamWr done as you said

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt
Comment (AdamWr) [inline AnnoyancesFilter/Cookies/sections/cookies_specific.txt:8061]:
```suggestion
denmulderboomteelt.com,tproger.ru###cookie-notice-wrapper
```

**Review by AdamWr — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt
Comment (Coolhabit) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt]: done

**Review by AdamWr — APPROVED** [review]

**Comment by Coolhabit** [comment]
@Alex-302 need your approve for merging

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**
